arches/app/media/js/bindings/datatable.js

```diff
@@ -27,7 +27,7 @@ define([
         init: function(element, valueAccessor, allBindingsAccessor, viewModel, bindingContext) {
             const options = ko.unwrap(valueAccessor());
             ko.unwrap(options.data);
-            if (valueAccessor().serverSide === true) {
+            if (options.dataTableOptions.serverSide === true) {
                 const table = $(element).closest('table').DataTable(valueAccessor().dataTableOptions);
```

Comment:
yeah this looks sane && matches the pattern in the rest of the document. However I see `valueAccessor()` being accessed directly here and on L36 -- can these be brought in-line with the current pattern?